helperFunctions/subscribeHelpers/InitiateSalesforceSessionHelpers/CheckChatStatusHelper.ts

```diff
@@ -172,4 +172,14 @@ export class CheckChatStatus {
 				return;
 			});
 	}
+
+	private async getAnalyticsQueueTime(read: IRead, assoc: RocketChatAssociationRecord) {
+		const { queueTimeStart } = await retrievePersistentData(read, assoc);
+		if (queueTimeStart === null) {
```

Comment:
using `===` here only checks for `null` and in the case of `undefined`, it will send the incorrect data. How about using `!queueTimeStart` ?